api/IDBRequest.json

```diff
@@ -182,6 +182,50 @@
               "standard_track": true,
               "deprecated": false
             }
+          },
+          "improved_reporting_large_value_read_failures": {
+            "__compat": {
+              "description": "Improved reporting for large value read failures",
+              "support": {
+                "chrome": [
+                  {
+                    "version_added": "132",
+                    "notes": "Throws a `NotReadableError` for unrecoverable large value read failures, and an `UnknownError` for transient large value read failures, with improved error messages."
+                  },
+                  {
+                    "version_added": "130",
+                    "partial_implementation": true,
+                    "notes": "Throws a `NotFoundError` for unrecoverable large value read failures, and a `DataError` for transient large value read failures."
```

Comment:
Note that the `NotFoundError` has been in the spec [since at least 2011](https://github.com/w3c/IndexedDB/commit/59e1598f73b871ff8c9eb76753e1c138784be59b#diff-d8adac750acc947123f2d8e77244ac97ed1e32fa65c0ca334c08fc620ab7ad60R1) (before it was referred to as` NOT_FOUND_ERR`).

Reply:
That's true. It is only mentioned here because Chrome reported that error type for read errors for a couple of versions (incorrectly, I assume), before it was then updated to `NotReadableError`.